azure-spring-boot-samples-4.x-pom.xml

```diff
@@ -135,181 +134,4 @@
     </plugins>
   </build>
 
-  <profiles>
-    <profile>
```

Comment:
The profiles buildpack and native should not be removed for Spring Cloud Azure 4.x, we need to keep them until the customer upgrades to Spring Boot 3.